README.md

```diff
@@ -1,6 +1,6 @@
-# <img src="https://user-images.githubusercontent.com/61895718/111565782-5ced2900-8772-11eb-9c43-c8801fc2a1a8.png" width="32px"> 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃𝟭.𝟳.𝟬
+# <img src="https://user-images.githubusercontent.com/61895718/111565782-5ced2900-8772-11eb-9c43-c8801fc2a1a8.png" width="32px"> 𝗔𝗯𝘆𝘀𝘀 𝗢𝘃𝗲𝗿𝗹𝗮𝘆 𝘃2.0.0
 
-![GitHub package.json version](https://img.shields.io/github/package-json/v/Chit132/abyss-overlay) ![GitHub all releases](https://img.shields.io/github/downloads/Chit132/abyss-overlay/total) [![Discord](https://img.shields.io/discord/822639649247592528?color=7289DA&label=Discord)](https://discord.gg/abyssoverlay)
+![GitHub package.json version](https://img.shields.io/github/package-json/v/Chit132/abyss-overlay) ![GitHub all releases](https://img.shields.io/github/downloads/Chit132/abyss-overlay/total) [![Discord](https://img.shields.io/discord/822639649247592528?color=7289DA&label=Discord)](https://discord.gg/eDnHTT3aBS)
```

Comment:
Can you please change the shields to this new version as it displays members not only online users per chit's request
![](https://dcbadge.vercel.app/api/server/eDnHTT3aBS?style=plastic)
https://dcbadge.vercel.app/api/server/eDnHTT3aBS?style=plastic